build.gradle

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+apply plugin: 'groovy'
+apply plugin: 'java'
+apply plugin: 'jacoco'
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven { url 'http://repo.jenkins-ci.org/releases/' }
+    maven { url 'http://repo.jenkins-ci.org/public/' }
+}
+
+dependencies {
+    compile group: 'org.codehaus.groovy', name: 'groovy-all', version: '3.0.3'
+    compile group: 'com.cloudbees', name: 'groovy-cps', version: '1.24'
+    compile 'org.kohsuke.stapler:stapler:1.257.2@jar'
+    compile 'org.jenkins-ci.main:jenkins-core:2.107.3@jar'
+    compile 'org.jenkins-ci.plugins.workflow:workflow-api:2.40@jar'
+    compile 'org.jenkins-ci.plugins.workflow:workflow-basic-steps:2.20@jar'
+    compile 'org.jenkins-ci.plugins.workflow:workflow-cps:2.81@jar'
+    compile 'org.jenkins-ci.plugins.workflow:workflow-cps-global-lib:2.16@jar'
+    compile 'org.jenkins-ci.plugins.workflow:workflow-step-api:2.22@jar'
+    compile group: 'com.lesfurets', name: 'jenkins-pipeline-unit', version: '1.1'
+
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.+'
+}
+
+sourceSets {
+    main {
+        groovy {
+            // all code files will be in either of the folders
+            srcDirs = ['src', 'vars', 'resources']
+        }
+    }
+    test {
+        groovy {
+            srcDirs = ['test', 'src', 'resources', 'vars']
+        }
+    }
+}
+
+jacocoTestReport {
+    reports {
+        xml.enabled false
+        csv.enabled false
+        html.enabled true
+    }
+}
+
+group = 'com.griddynamics.devops'
+description = 'Modular Pipeline Library'
+sourceCompatibility = '1.8'
+
+task prepareForTests(type: Copy) {
+    from(projectDir) {
+        include 'src/**/*.*'
+        include 'vars/**/*.*'
+        include 'resources/**/*.*'
+        include 'build.gradle'
+        include 'settings.gradle'
+    }
+    // TODO: try to come up with a more elegant way of doing this - it's like that to avoid completely breaking maven support
+    into "${buildDir}/classes/groovy/test/${project.name}@snapshot"
+}
+
+test {
+    dependsOn prepareForTests
+    // using grape causes issues with missing classpaths
+    // as gradle does not use ivy by default.
+    systemProperty 'groovy.grape.enable', 'false'
```

Comment:
Hmmm, I thought maven allows to use grapes and they could be used by tests sometimes, maybe to figure out what's wrong with it instead of just disabling? Maybe it's just your env issue?